test_examples/multiple_includes/uses_terraform_12/terragrunt.hcl

```diff
@@ -0,0 +1,11 @@
+include {
+  path = find_in_parent_folders("use_terraform_12_parent.hcl")
+}
+
+terraform {
+  source = "git::git@github.com:transcend-io/terraform-aws-fargate-container?ref=v0.0.4"
+}
+
+inputs = {
+  foo = "bar"
+}
```

Comment:
your editor doesn't like newlines at the EOF? 🙃

Reply:
🇺🇸  Vim gives me the freedom to decide 🇺🇸